homeassistant/components/bluesound/media_player.py

```diff
@@ -314,6 +315,15 @@ def __init__(
 
         self._init_callback = init_callback
 
+        self._attr_unique_id = format_unique_id(sync_status.mac, port)
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, format_unique_id(sync_status.mac, port))},
+            name=sync_status.name,
```

Comment:
Can the name of the sync status be different per virtual player?

Reply:
I can only guess. But the docs seem to suggest “yes”.